policy/release/cve_test.rego

```diff
@@ -52,6 +73,28 @@ test_success_with_rule_data if {
 		with data.rule_data.restrict_cve_security_levels as ["spam"]
 }
 
+test_success_with_rule_data_deprecated_name if {
+	slsav1_task_with_result := tkn_test.slsav1_task_result_ref(
+		"clair-scan",
+		[{
+			"name": cve._deprecated_result_name,
```

Comment:
Testing the success case is good. But I think we also want to make sure a violation/warnings happens when the deprecated name is used as well.